src/Xamarin.Android.Build.Tasks/Utilities/MarshalMethodsAssemblyRewriter.cs

```diff
@@ -95,10 +123,161 @@ void MoveFile (string source, string target)
 				Files.CopyIfChanged (source, target);
 				try {
 					File.Delete (source);
-				} catch (Exception ex) {
+				} catch (Exception) {
 					log.LogWarning ($"Unable to delete source file '{source}' when moving it to '{target}'");
 				}
 			}
+
+			string ToStringOrNull (object? o)
+			{
+				if (o == null) {
+					return "'null'";
+				}
+
+				return o.ToString ();
+			}
+		}
+
+		MethodDefinition GenerateBlittableWrapper (MarshalMethodEntry method, Dictionary<AssemblyDefinition, CustomAttribute> unmanagedCallersOnlyAttributes)
+		{
+			Console.WriteLine ($"\t  Generating blittable wrapper for: {method.NativeCallback.FullName}");
```

Comment:
s/Console.WriteLine/Log.LogDebugMessage/g

Reply:
Most of these will go away in the next PR